frontend/services/src/events/eventsService.ts

```diff
@@ -254,7 +252,7 @@ export async function deleteEventByName(eventName: string): Promise<void> {
     querySnapshot.forEach(async (eventDoc) => {
       await deleteDoc(eventDoc.ref);
     });
-    eventServiceLogger.info(`Deleting Event by name successfully ${eventName}`);
+    eventServiceLogger.info(`Deleting Event by Name successfull ${eventName}`);
```

Comment:
it would be more useful to log the eventId for each of these.

I think a lot of these console log statements have been put here originally for coding and debugging purposes.